src/Bicep.Cli/Services/PlaceholderParametersWriter.cs

```diff
@@ -10,11 +10,11 @@
 
 namespace Bicep.Cli.Services
 {
-    public class ParametersFileWriter
+    public class PlaceholderParametersWriter
```

Comment:
>PlaceholderParametersWriter [](http://example.com/codeflow?start=17&length=27)

What's the difference between the ParametersWriter and PlaceholderParametersWriter?